python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -270,19 +283,15 @@ def assert_corrections(self, mode):
         IOError
             If mode is not enabled or disabled
         """
-        if mode not in ('enabled', 'disabled'):
-            raise IOError("Mode must be either enabled or disabled")
+        if mode > 1:
```

Comment:
This should be `if mode not in (0, 1)` (or ditch this test if you make the argument boolean)